### PR TITLE
Update logging documentation

### PR DESCRIPTION
**Pull request by MSathieu**
Include documentation on how to disable the logging.
#2029 @bf4

### CODE REVIEW

File: docs/general/logging.md [hunk 1]
```diff
@@ -12,3 +12,9 @@ You may customize the logger in an initializer, for example:
 ```ruby
 ActiveModelSerializers.logger = Logger.new(STDOUT)
 ```
+
+You can also disable the logger, just put this in `config/application.rb`:
+
+```ruby
+ActiveSupport::Notifications.unsubscribe(ActiveModelSerializers::Logging::RENDER_EVENT)
```
Comment (bf4) [inline docs/general/logging.md]: It's a better practice to put this in an initializer in a Rails app.

`config/initializers/active_model_serializers.rb` is the usual location.

```ruby
require 'active_model_serializers'
ActiveSupport::Notifications.unsubscribe(ActiveModelSerializers::Logging::RENDER_EVENT)
```

and perhaps reference the relevant railtie https://github.com/rails-api/active_model_serializers/blob/0-10-stable/lib/active_model_serializers/railtie.rb#L23-L27 ?

**Comment by bf4** [comment]
@MSathieu Thanks so much for this and the quick turnaround!

File: docs/general/logging.md (hunk 1 above)
Comment (MSathieu) [inline docs/general/logging.md]: Why requiring it

**Comment by MSathieu** [comment]
@bf4 Updated it to use an initializer

File: docs/general/logging.md (hunk 1 above)
Comment (bf4) [inline docs/general/logging.md]: @MSathieu not exactly. It's actually the call to `Bundle.require` at the top of most `config/application.rb` and then only if the gem doesn't have `require: false`.

Also, relying on autoloading of your project's classes is pretty much required for code reloading to work. But that doesn't really apply to gems.  It's a good idea to require a gem before you use it. But this opinion is controversial enough that I won't tell you to include it :)

**Comment by MSathieu** [comment]
@bf4 I have updated it to require active_model_serializers in the initializer

**Comment by bf4** [comment]
@MSathieu would you mind adding a changelog entry for yourself under 'misc'?

**Comment by MSathieu** [comment]
@bf4 Done

**Pull request merged**